mobile/src/global.css

```
@@ -0,0 +1,9 @@
+:root {
+  --font-display:
+    Spline Sans, Inter, ui-sans-serif, system-ui, sans-serif, Apple Color Emoji, Segoe UI Emoji,
```

Stylelint (17.14.0)

[error] Expected "Spline" to be "spline" (value-keyword-case)

(value-keyword-case)